### PR TITLE
feat: /models endpoint

### PR DESCRIPTION
**Pull request by sammcj**
Add /models endpoint that just proxies the response from upstream

### CODE REVIEW

File: optillm.py [hunk 1]
```diff
@@ -97,8 +97,7 @@ def proxy():
                 break
         else:
             # If no known approach is found in the model name, default to 'bon'
-            approach = 'bon'
-
+            approach = "bon"
```
Comment (sammcj) [inline optillm.py:100]: Oh whoops, sorry I'll fix this.

**Comment by codelion** [comment]
LGTM, Thank you!

**Comment by codelion** [comment]
@sammcj I recently added a CLA, can you please sign it here - https://cla-assistant.io/codelion/optillm

**Comment by sammcj** [comment]
Yeah no worries, done

**Comment by codelion** [comment]
Thank you.

**Pull request merged**